provisiondmarcimport_policy.tf

```diff
@@ -102,7 +103,6 @@ data "aws_iam_policy_document" "provisiondmarcimport" {
       "logs:DeleteResourcePolicy",
       "logs:DeleteRetentionPolicy",
       "logs:ListTagsLogGroup",
-      "logs:PutResourcePolicy",
```

Comment:
I suspect that `logs:PutResourcePolicy` could come back to live here if we changed `arn:aws:logs:${var.aws_region}:${local.dns_account_id}:log-group:/aws/aes/domains/${var.elasticsearch_domain_name}/application-logs*` to `arn:aws:logs:${var.aws_region}:${local.dns_account_id}:log-group:/aws/aes/domains/${var.elasticsearch_domain_name}/application-logs:*` and `arn:aws:logs:${var.aws_region}:${local.dns_account_id}:log-group:/aws/lambda/${var.lambda_function_name}*` to `arn:aws:logs:${var.aws_region}:${local.dns_account_id}:log-group:/aws/lambda/${var.lambda_function_name}:*` in the `resources` block.

Reply:
Good call- I tested this and you are correct.  I made the changes in https://github.com/cisagov/cool-dns-dmarc-import/pull/3/commits/bdbc617df2cd6771607f320e14318e72c5d5d77e and applied them in Production.  I was able to verify that data continued to flow into the log groups and the count of documents in the dmarc reports DB continued to increase.